test/objects.js

```diff
@@ -930,9 +930,9 @@
         'an ArrayBuffer': buffer,
         'a TypedArray': new Uint8Array(buffer)
       };
-      // Some older browsers support typed arrays but not DataView.
-      if (typeof DataView !== 'undefined') {
-        checkValues['a DataView'] = new DataView(buffer);
+      var DataViewImpl = typeof NativeDataView !== 'undefined' ? NativeDataView : typeof DataView !== 'undefined' ? DataView : undefined; // Browser could override DataView
```

Comment:
This is a repetition of line 612. You can lift it to the scope of the IIFE wrapper (but first change the comment as I suggested there).